pytorch_lightning/callbacks/quantization.py

```diff
@@ -188,7 +189,11 @@ def on_fit_start(self, trainer, pl_module):
             if self._observer_type == "histogram":
                 pl_module.qconfig = torch.quantization.get_default_qconfig(self._qconfig)
             elif self._observer_type == "average":
-                pl_module.qconfig = torch.quantization.get_default_qat_qconfig(self._qconfig)
+                # Needed because of https://github.com/pytorch/pytorch/issues/64564
+                if _TORCH_GREATER_EQUAL_1_10:
+                    pl_module.qconfig = torch.quantization.get_default_qat_qconfig(self._qconfig, None)
```

Comment:
Could you add a comment for what None corresponds to here as well? It'd save readers one hop of going through the quantization APIs